Discussion initial draft (#661)
greenelab/covid19-review: content/50.discussion.md

Efforts to identify therapeutic treatments to treat patients early in the course of infection have been more ambiguous.
Early interest in the drugs hydroxychloroquine and chloroquine yielded no promising results from studies with robust experimental designs.
The experimental drug remdesivir, which was developed for ebola, has received enough support from early analyses to receive FDA approval, but results have been mixed.
The potential for other drugs, such as tocilizumab, to reduce recovery time remains unclear, but some early results were promising.

### Additional Therapeutics of Interest

Given what is currently known about these therapeutics for COVID-19, a number of related therapies beyond those explored above may also prove to be of interest.
For example, the demonstrated benefit of dexamethasone and the ongoing potential of tocilizumab for treatment of COVID-19 suggests that other anti-inflammatory agents might also hold value for the treatment of COVID-19.

> Given that current evidence about treating COVID-19 with dexamethasone suggests that the need to curtail the “cytokine storm” inflammatory response to the virus can transcend the risks of immunosuppression, exploration of more anti-inflammatory agents may be warranted.

Author: ```suggestion
Given that current evidence about treating COVID-19 with dexamethasone suggests that the need to curtail the cytokine storm inflammatory response to the virus can transcend the risks of immunosuppression, exploration of more anti-inflammatory agents may be warranted.
```

While dexamethasone is considered widely available and generally affordable, the high costs of biologics such as tocilizumab therapy may present obstacles to wide-scale distribution of this drug if it proves of value.
At the doses used for rheumatoid arthritis patients, the cost for tocilizumab ranges from $179.20 to $896 per dose for the IV form and $355 for the pre-filled syringe [@url:https://www.ncbi.nlm.nih.gov/books/NBK349513/table/T43/].

> There are several anti-inflammatory agents used for the treatment of autoimmune diseases that may also be able to counter the effects of the “cytokine storm” induced by the virus, some of which, such as cyclosporine, are likely to be more cost-effective and readily available than biologics [@url:https://escholarship.umassmed.edu/meyers_pp/385].

Author: ```suggestion
There are several anti-inflammatory agents used for the treatment of autoimmune diseases that may also be able to counter the effects of the cytokine storm induced by the virus, some of which, such as cyclosporine, are likely to be more cost-effective and readily available than biologics [@url:https://escholarship.umassmed.edu/meyers_pp/385].
```

In mouse models, poly(I:C) and CpG, which are agonists of toll-like receptors TLR3 and TLR9, respectively, showed protective effects when administered prior to SARS-CoV infection [@doi:10.1128/JVI.01410-12].
Therefore, TLR agonists hold some potential for broad-spectrum prophylaxis.

Given that a large number of clinical trials are currently in progress, more information about the potential of these and other therapeutics should become available over time.
This information, combined with advances in understanding the molecular structure and viral pathogenesis of SARS-CoV-2, may lead to a more complete understanding of how the virus affects the human host and what strategies can improve outcomes.
To date, investigations of potential therapeutics for COVID-19 have focused primarily on repurposing existing drugs.
This approach is necessary given the urgency of the situation as well as the extensive time required for developing and testing new therapies.
However, in the long-term, new drugs specific for treatment of COVID-19 may also enter development.
Development of novel drugs is likely to be guided by what is known about the pathogenesis and molecular structure of SARS-CoV-2.

> For example, understanding the structures of SARS-CoV-2 components may allow for the development of small molecule inhibitors of those components.

Reviewer: ```suggestion
For example, understanding the various structural components of SARS-CoV-2 may allow for the development of small molecule inhibitors of those components.
```

Much work remains to be done to determine further crystal structures of other viral components, understand the relative utility of targeting different viral components, perform additional small molecule inhibitor screens, and determine the safety and efficacy of the potential inhibitors.
While still nascent, work in this area is promising.
Over the longer term, this approach and others may lead to the development of novel therapeutics specifically for COVID-19 and SARS-CoV-2.

### Social Factors Influencing COVID-19 Outcomes

In addition to understanding the fundamental biology of the SARS-CoV-2 virus and COVID-19, it is critial to consider how the broader environment can influence both COVID-19 outcomes and efforts to develop and implement treatments for the disease.
The evidence clearly indicates that social environmental factors are critical determinants of individuals' and communities' risks related to COVID-19.

> There are distinct components to COVID-19 susceptibility, and an individual's risk can be elevated at one or all stages from exposure to recovery/mortality: an individual may be more likely to be exposed to the virus, more likely to get infected once exposed, more likely to have serious complications once infected, be less likely to receive adequate care once they are seriously ill.

Reviewer: ```suggestion
There are distinct components to COVID-19 susceptibility, and an individual's risk can be elevated at one or all stages from exposure to recovery/mortality: an individual may be more likely to be exposed to the virus, more likely to get infected once exposed, more likely to have serious complications once infected, and be less likely to receive adequate care once they are seriously ill.
```

Another potential approach is the development of antivirals, which could be broad-spectrum, specific to coronaviruses, or targeted to SARS-CoV-2.
Development of new antivirals is complicated by the fact that none have yet been approved for human coronaviruses.
Intriguing new options are emerging, however.
Beta-D-N4-hydroxycytidine (NHC) is an orally bioavailable ribonucleotide analog showing broad-spectrum activity against RNA viruses, which may inhibit SARS-CoV-2 replication _in vitro_ and _in vivo_ in mouse models of HCoVs [@doi:10.1126/scitranslmed.abb5883].
A range of other antivirals are also in development.
Development of antivirals will be further facilitated as research reveals more information about the interaction of SARS-CoV-2 with the host cell and host cell genome, mechanisms of viral replication, mechanisms of viral assembly, and mechanisms of viral release to other cells; this can allow researchers to target specific stages and structures of the viral life cycle.
Many researchers have also focused their attention to the potential use of dietary supplements and nutraceuticals.

> Indeed, there has been recent interest for the potential use of vitamin D as a prophylactic and therauetic agent against COVID-19 as several observational studies have linked low vitamin D status to its incidence [@doi:10.1111/joim.13149; @doi10.1001/jamanetworkopen.2020.19722]. 

Reviewer: ```suggestion
Indeed, there has been recent interest for the potential use of vitamin D as a prophylactic and therauetic agent against COVID-19 as several observational studies have linked low vitamin D status to its incidence [@doi:10.1111/joim.13149; @doi:10.1001/jamanetworkopen.2020.19722]. 
```
